localstack/services/events/target.py

```diff
@@ -33,6 +49,38 @@ def transform_event_with_target_input_path(
     return formatted_event
 
 
+def get_template_replacements(
+    input_transformer: InputTransformer, event: FormattedEvent
+) -> dict[str, Any]:
+    """Extracts values from the event using the input paths map keys and places them in the input template dict."""
+    template_replacements = {}
+    transformer_path_map = input_transformer.get("InputPathsMap", {})
+    for placeholder, transformer_path in transformer_path_map.items():
+        if placeholder in PREDEFINED_PLACEHOLDERS:
+            continue
+        value = extract_jsonpath(event, transformer_path)
```

Comment:
Can this fail with an invalid jsonpath?

Reply:
the boto client validation of input template that checks that only placeholders that are using a json path that is present in the defined event should handle errors long before `extract_jsonpath` will be called, therefore it should be save.